### DISCUSSION TITLE
Alias command with parameter

### DISCUSSION THREAD

**Discussion by satsumah** [jagrosh/MusicBot#1163] [Ideas]
I might be asking for something out of scope here, so feel free to roast/flame/report to the cyber police if I'm asking something out of pocket.

What I'm looking for is something where I can type, for example, !rickroll and the alias is interpreted as "!play https://www.youtube.com/watch?v=dQw4w9WgXcQ." Is that possible in config? And if not, would this be possible with some light edits to source?

**Comment by MichailiK** [comment]
There already exists a feature request for "macros": #483

**Reply by satsumah** [reply]
Oh hey nice, that's pretty much exactly what I was looking for. Thank you!
